source/stylesheets/refills/unstyled/_scroll-on-page.scss

```diff
@@ -0,0 +1,26 @@
+.scroll-on-page {
+  $large-screen: em(860) !default;
+
+  aside {
+    @include media($large-screen) {
+      @include span-columns(4);
+    }
+  }
+
+  .scroll-on-page-link {
+    display: block;
+    outline: none;
+  }
+
+  article {
+    section p:last-of-type {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Selector should have depth of applicability no greater than 2, but was 4